lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/file/DBOMultipartUpload.java

```diff
@@ -221,6 +239,14 @@ public void setUploadToken(String uploadToken) {
 		this.uploadToken = uploadToken;
 	}
 

```

Comment:
add the new table to getSecondaryTypes()

	list.add(new DBOMultipartUploadComposerPartState());